adding blog post for first touch video (#608)
k8ssandra/k8ssandra: docs/content/en/blog/articles/k8ssandra-first-touch/index.md

---
date: 2021-03-30
title: "K8ssandra First Touch"
linkTitle: "K8ssandra First Touch"
description: >
  We've updated our approach to managing the K8ssandra roadmap and would love to get your input.
author: Aleks Volochnev ([@hadesarchitect](https://twitter.com/hadesarchitect))
---

> This live demo shows you how to run an Apache Cassandra® cluster in Kubernetes using K8ssandra. Steps include: installing a cluster using Helm, configuring ingress using Traefik, checking that the installation is running correctly, explanation of each pod included in K8ssandra, viewing metrics using Prometheus and Grafana, and running repairs with Reaper. 

Reviewer: ```suggestion
This walkthrough shows how to run an Apache Cassandra® cluster in Kubernetes using K8ssandra. 

Steps include:

* Installing a cluster using Helm
* Configuring ingress with Traefik
* Validating the installation
* Exploration of all pods
* Viewing metrics using Prometheus and Grafana
* Running repairs with Reaper
```

For more information on installing K8ssandra and first use, check out the [getting started guide](/docs/getting-started/).

{{< youtube id="qlZVLEWzJq0" >}}

> 
> 

Reviewer: ```suggestion
```

Is all this whitespace necessary?
